contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -288,3 +288,5 @@ The [upstream changelog](https://github.com/bitnami/charts/tree/master/bitnami/r
 22.0.0 upgrades ClickHouse from version `21.6.5` to `22.3.6.5`. This update brings several improvements to the overall service. For more info, you can look at the [upstream changelog](https://clickhouse.com/docs/en/whats-new/changelog/#clickhouse-release-v223-lts-2022-03-17).
 
 Note: the ClickHouse pod(s) will be reprovisioned as part of this upgrade. We expect no downtime for the ingestion pipeline.
+
+Note: Async migration 0004 is required and could cause problems if not done completed. If you already the upgrade just run the migration now.
```

Comment:
I don't get how this is related to the ClickHouse upgrade

Reply:
It's not, the problem is I don't know when it became required. In the summary:
> Don't want to spend too much time investigating/fixing here, let's just call it out, because I believe most folks have ran the 0004 migration already & running it later seemed to work quite well.